packages/elements/src/number-field/__test__/number-field.step.test.js

```diff
@@ -220,4 +220,39 @@ describe('number-field/Step', function () {
       await expectValues(el, [-1, -2, -3, -4, -5], UP, -1.5);
     });
   });
+
+  describe('Long press spinner', function () {
+    const dispatchLongTapEvent = async (el, wait = 1000) => {
+      setTimeout(() =>
+        el.dispatchEvent(
+          new Event('tapstart', {
+            bubbles: true
+          })
+        )
+      );
+      await oneEvent(el, 'tapstart');
+      if (wait) {
+        await aTimeout(wait);
+      }
+      setTimeout(() =>
+        el.dispatchEvent(
+          new Event('tapend', {
+            bubbles: true
+          })
+        )
+      );
+      await oneEvent(el, 'tapend');
+    };
+
+    it('By tap, Step Up', async function () {
+      const el = await fixture('<ef-number-field></ef-number-field>');
+      await dispatchLongTapEvent(el.spinnerUpEl);
+      expect(Number(el.value)).to.greaterThan(3);
```

Comment:
What does `3` number means ?

Reply:
number 3 is a magic number to assume that long press is working

Reply:
In that case, wouldn't `greater than 1` already proves the functionality.

Reply:
Maaaaake sense, agree.

Reply:
On this point, I think he want to check the value should increase to `3` if user press spinner for 1 second.

Reply:
I think the exact value is unpredictable and depend to test device performance. but at least pressing spinner for one second, the value should literally greater than 1.

Reply:
If so, we have could have 2 test cases. One just to test long press functionality. Another is to test incremental speed for long press. I feel the later test case is unnecessary, but could be added just in case.

Reply:
Agreed that '1' is enough to be a value for test. So, I updated.